src/app/organizations/organization-form/organization-form.component.html

```diff
@@ -1,11 +1,22 @@
 <form (submit)="save()" [formGroup]="form" class="grid formgrid p-fluid py-2" novalidate>
+  <div *ngIf="new" class="col-12 mb-2">
+    <p-message i18n-text severity="info" text="The name of an organization can not be changed later."></p-message>
+  </div>
   <div *ngIf="new" class="field mb-4 col-12">
-    <label class="font-medium text-900" for="name">Name</label>
+    <label class="required" for="name">Name</label>
     <input formControlName="name" id="name" pInputText type="text" />
+    <small *ngIf="form.get('name')?.hasError('alreadyExists')" class="block p-error" i18n>
+      An organization with the same name already exists.
+    </small>
+    <small *ngIf="form.get('name')?.hasError('pattern')" class="block p-error" i18n>
+      The organization name must match the pattern
+      <code>(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?</code>
```

Comment:
For the input "Org1" I got the following backend response:

```
{
        "reason": "FieldValueInvalid",
        "message": "Invalid value: \"Org1\": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')",
        "field": "metadata.name"
      }
```

So, I assume this should be the correct pattern: `[a-z0-9]([-a-z0-9]*[a-z0-9])?')`

"my_org" does also not trigger an error the frontend validation.

Also, the backend error message doesn't match the regex: "...and must start and end with an alphanumeric character", but the regex allows to start with 0-9 as well.